pages/common/cpio.md

```diff
@@ -1,16 +1,15 @@
 # cpio
 
-> Archiving utility.
+> Copies files to and from archives.
```

Comment:
I read that (and appreciated the effort), but didn't realize this wording fitted your perception of "weird". To me it sounds like perfectly reasonable (if a bit techy) English.